airship/AIR-141_runtimeclass.md

```diff
@@ -8,3 +8,84 @@
 
 Jira issues:
 - [RunTimeClass configuration](https://airship.atlassian.net/browse/AIR-141)
+
+## RuntimeClass Introduction
+It is possible to define RunTimeClass kind configuration to control runtimes used for pods.
+See example runtimeclass definition below.
+```
+apiVersion: node.k8s.io/v1beta1
+kind: RuntimeClass
+metadata:
+  name: containerdruntimeclass
+handler: containerd
+```
+After class is configured, it can be taken into use in pods, see example pod definition below.
+```
+apiVersion: v1
+kind: Pod
+metadata:
+  name: example
+spec:
+  runtimeClassName: containerdruntimeclass
+  containers:
+    - name: dummy-pod
+      image: ubuntu
+      command: ["/bin/bash", "-ec", "while :; do echo '.'; sleep 5 ; done"]
+  restartPolicy: Never
+```
+# Runtime selection for Kubeadm
+Kubeadm detects available runtime sockets and if there is more than one runtime socket available, kubeadm expects
+that user will define wanted runtime by setting --cri-socket parameter.
+After Kubelet confiuration and alternative runtime installations (See examples below), kubeadm can be executed
+as part of cloud init.
+```
+kubeadm init --cri-socket /var/run/crio/crio.sock
```

Comment:
Also possible to do it from kubeadm config file

apiVersion: kubeadm.k8s.io/v1beta2
kind: InitConfiguration
localAPIEndpoint:
  advertiseAddress: 192.168.10.2
  bindPort: 6443
nodeRegistration:
  criSocket: /var/run/dockershim.sock
  name: mycluster1-control-plane
  taints:
  - effect: NoSchedule
    key: node-role.kubernetes.io/master
---